src/monovar_wrapper.sh

```diff
@@ -1,2 +1,6 @@
 #!/bin/bash
+source activate monovar
```

Comment:
Again, Can we use the conda environment feature? https://snakemake.readthedocs.io/en/stable/snakefiles/deployment.html#combining-conda-package-management-with-containers

Reply:
In the original version of scsim, it uses source activate so I just follow the style. Sure, I can change to conda activate.